components/automate-ui/src/app/pages/create-notification-modal/create-notification-modal.component.ts

```diff
@@ -41,7 +42,7 @@ enum UrlTestState {
   templateUrl: './create-notification-modal.component.html',
   styleUrls: ['./create-notification-modal.component.scss']
 })
-export class CreateNotificationModalComponent implements OnInit {
+export class CreateNotificationModalComponent implements OnInit, OnDestroy {
   @Input() openEvent: EventEmitter<boolean>;
```

Comment:
Just need to update this to match the earlier change:
```suggestion
  @Input() openEvent: EventEmitter<void>;
```

Reply:
I have added these changes and updated the PR.